helix-term/src/commands.rs

```diff
@@ -2608,26 +2612,15 @@ pub mod insert {
         {
             // TODO: what if trigger is multiple chars long
             let is_trigger = triggers.iter().any(|trigger| trigger.contains(ch));
+            // lsp doesn't tell us when to close the signature help, so we request
+            // the help information again after common close triggers which should
+            // return None, which in turn closes the popup.
+            let close_triggers = &[')', ';', '.'];
```

Comment:
Would this be language specific?

Reply:
I don't think it's going to make much difference here since common closing characters are included here. We can always expand this list here with a few additional characters if we need to.